libraries/aws/shadow/cbmc/proofs/Shadow_GetTopicString/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=Shadow_GetTopicString_harness
+
+DEFINES +=
+INCLUDES +=
+
+# The thing name field length is bounded.
+MAX_THINGNAME_FIELD_LENGTH=128
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET += strncmp.0:10
+UNWINDSET += __CPROVER_file_local_shadow_c_validateThingName.0:$(MAX_THINGNAME_FIELD_LENGTH)
```

Comment:
128 might be an excessive length and a long time to loop. How long the proof take to run?

Reply:
about 400 second.

Reply:
That is kind of long maybe we can make MAX_THINGNAME_FIELD_LENGTH to 10?

Reply:
please review [this commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1134/commits/87cb11ae1496f618c50d49ad2ed7fd5c3630a3ec) for the update.